modules/gentemplates/openapi-generator/src/main/java/com/reprezen/genflow/openapi/generator/OagCodegenGenTemplateBase.java

```diff
@@ -64,6 +64,32 @@ public void configure() throws GenerationException {
 				"Each property should be a JSON object with a name/value pair for each property.",
 				"Example: for '-Dmodels -Dapis=User,Pets' use the following:", "value:", "  models: ''",
 				"  apis: Users,Pets"));
+		define(parameter().named(LANGUAGE_SPECIFIC_PRIMITIVES).optional().withDescription(
+				"Specifies additional language specific primitive types.",
+				"You can have multiple occurrences of this option.", //
+				"Example: -Pet", //
+				"         -User"));
+		define(parameter().named(TYPE_MAPPINGS).optional().withDescription(
+				"Sets mappings between Swagger spec. types and generated code types.", //
+				"You can have multiple occurrences of this option.", //
+				"Example: array:List", //
+				"         map:Map"));
+		define(parameter().named(INSTANTIATION_TYPES).optional().withDescription(
+				"Sets mappings between Swagger spec. types and generated code instance types.",
+				"You can have multiple occurrences of this option.", //
+				"Example (in Java): array:ArrayList", //
+				"                   map:HashMap"));
+		define(parameter().named(IMPORT_MAPPINGS).optional().withDescription(
+				"Specifies mappings between a given class and the import that should be used for that class.",
+				"You can have multiple occurrences of this option.", //
+				"Example: ArrayList:java.util.ArrayList", //
+				"		     HashMap:java.util.HashMap", //
+				"		     User:com.mycomp.User"));
+		define(parameter().named(RESERVED_WORDS_MAPPINGS).optional().withDescription(
+				"Specifies how a reserved name should be escaped to. Otherwise, the default _<name> is used.",
```

Comment:
"Specifies a mapping between reserved words in the target language and legal, non-reserved names.",
"Where the OpenAPI document uses a reserved word as a type, property operation or parameter name",  
"the generator will substitute the name provided in the map. Otherwise, the default _<name> is used.",